File: trabajos encargados/trabajo encargado semana 1 condicional -- bucles/ejercicios condicionales/ejer1condicional.cpp
#include <iostream>
using namespace std;
int main(){
    cout << "ingrese numero: ";
    int numero1;
    cin >> numero1;

    if (numero1 < 0){
        cout << "el numero es invalido solo positivos: " << endl;
    }else{
        cout << "el numero es correcto: " << endl;
    }if(numero1 % 2 == 0 ){
        cout << "el numero: " << numero1 << " es par:" << endl;
    }else if(numero1 % 2 == 1){
        cout << "el numero: " << numero1 << " es impar:" << endl;
    }
    return 0 ;
}

File: trabajos encargados/trabajo encargado semana 1 condicional -- bucles/ejercicios condicionales/ejer1condicional.py
numero1 = int(input("Ingrese numero: "))

if numero1 < 0:
    print("El numero es invalido, solo positivos")
else:
    print("El numero es correcto")

if numero1 % 2 == 0:
    print(f"El numero {numero1} es par")
else:
    print(f"El numero {numero1} es impar")
